core/src/main/java/tc/oc/pgm/map/MapInfoImpl.java

```diff
@@ -367,10 +370,13 @@ private static class VariantData implements VariantInfo {
     private final String mapName;
     private final String mapId;
     private final String world;
+    private final Range<Version> serverVersions;
 
     public VariantData(Element root, @Nullable Element variantEl) throws InvalidXMLException {
       String name = assertNotNull(Node.fromRequiredChildOrAttr(root, "name").getValueNormalize());
       String slug = assertNotNull(root).getChildTextNormalize("slug");
+      Node minVer = Node.fromAttr(root, "min-server-version");
```

Comment:
Modern (1.9+) worlds have got version metadata in `level.dat`, maybe the minimum server version could be inferred from there if not specified?

Reply:
Having this in the XML makes it more visible to mapdevs, and the behavior more predictable. Additionally, there's times where a map may be made in 1.9 or 1.12 and yet be loaded and play in 1.8 just fine. Forcibly making those not load would be a behavior change.

I'm fine with having a default min-server-version of 1.13 if the world is in 1.13+ format, as that is never going to be able to load in 1.8, but otherwise keep it xml defined

Reply:
> I'm fine with having a default min-server-version of 1.13 if the world is in 1.13+ format, as that is never going to be able to load in 1.8, but otherwise keep it xml defined

This seems ideal and aligns similarly with how PGM tends to treat `level.dat` information at present.